inference-engine/tests/ie_test_utils/common_test_utils/common_utils.hpp

```diff
@@ -47,6 +47,21 @@ inline std::string vec2str(const std::vector<vecElementType> &vec) {
     return std::string("()");
 }
 
+inline std::string pair2str(const std::pair<size_t, size_t>& p) {
+    std::ostringstream result;
+    result << "(" << p.first << "." << p.second << ")";
+    return result.str();
+}
+
+template<>
```

Comment:
why `template<>` is here? It is not necessary. And remove extra functions (from added) according my previous comments

Reply:
`template<>` was removed, and what did you mention with extra functions? Could you explain more about it?